mobile.html

```diff
@@ -0,0 +1,179 @@
+                    </div>
+                    <div class="frame-cont">
+                        <div class="frame-lang1">
+                            <p>Bootstrap</p>
+                        </div>
+                        <div class="frame-lang2">
+                            <p>Ruby on Rails</p>
+                        </div>
+                        <div class="frame-lang3">
+                            <p>Rspec</p>
+                        </div>
+                        <div class="frame-lang4">
+                            <p>CapyBara</p>
+                        </div>
+                        <div class="frame-lang5">
+                            <p>Selenuim</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="my-skills">
+                    <div class="skill-name">
+                        <p>Skills</p>
+                    </div>
+                    <div class="skill-cont">
+                        <div class="skill1">
+                            <p>Codekit</p>
+                        </div>
+                        <div class="skill2">
+                            <p>GitHub</p>
+                        </div>
+                        <div class="skill3">
+                            <p>Codepen</p>
+                        </div>
+                        <div class="skill4">
+                            <p>GitLab</p>
+                        </div>
+                        <div class="skill5">
+                            <p>Terminal</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="underline2">
+                    <div class="line"></div>
+                </div>
+                <div class="social-media">
+                    <div class="media">
+                        <div class="social1">
+                            <img src="icons/Vector5.png" alt="">
+                        </div>
+                        <div class="social2">
+                            <img src="icons/Vector4.png" alt="">
+                        </div>
+                        <div class="social3">
+                            <img src="icons/Vector3.png" alt="">
+                        </div>
+                        <div class="social4">
+                            <img src="icons/Vector2.png" alt="">
+                        </div>
+                        <div class="social5">
+                            <img src="icons/Vector1.png" alt="">
+                        </div>
+                    </div>
+                    <div class="work-btn">
+                        <div class="arrowdown">
+                            <img src="icons/Group55.png" alt="">
+                        </div>
+                        <div class="work">
+                            <p>Work</p>
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] The section is expected to have a proper background image just as the one provided in the Figma design.